simulation_parameters/Constants.cs

```diff
@@ -774,6 +775,14 @@ public static class Constants
         "user://mods",
     };
 
+    /// <summary>
+    ///   Regex expresions to categorize different file types.
+    /// </summary>
+    public static readonly Regex BackupRegex = new(@"^.*\.backup\.thrivesave$");
```

Comment:
This still doesn't use the other constant for thrivesave part.

Reply:
My bad, I got caught up in working on other things.